src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -446,6 +447,63 @@ public void CleanupAfterEachTest()
             }
         }
 
+        /// <summary>
+        /// Checks if the connection string provided in the config is correctly updated for MSSQL.
```

Comment:
Can you explicitly state that MySQL, Cosmos, and PG do not support this?

- Also, in your PR description, can you explain why the lack of support? PG has support for https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS:~:text=Chapter%C2%A020.-,application_name,Specifies%20a%20value%20for%20the%20application_name%20configuration%20parameter.,-fallback_application_name

Asking to see if it is a quick win here to avoid feature drift between database types.

Reply:
Yes, PGsql supports application name, but we need to check how that information can be used for telemetry, can be a seperate PR.

Reply:
can you open an issue for that?